docs/user-manual/en/paging.md

```diff
@@ -171,6 +171,13 @@ Once that limit is reached any message will be blocked. (unless the protocol
 doesn't support flow control on which case there will be an exception thrown
 and the connection for those clients dropped).
 
+## Page Sync Timeout
+
+The pages are synced periodically and the sync period is configured through
```

Comment:
can you change the CLI create to perform an NIO test.. and set this value with the same amount as NIO.

if NIO, this value should be the same .. if AIO a separate test should be done, and then applied to page-sync timeout.

Reply:
I pushed a new commit that changes the CLI create to autotune page-sync-timeout too.